lib/util/fs/index.ts

```diff
@@ -162,3 +165,12 @@ export function localPathIsFile(pathName: string): Promise<boolean> {
     .then((s) => s.isFile())
     .catch(() => false);
 }
+
+export async function globLocalFiles(globPattern: string): Promise<string[]> {
+  const { localDir } = GlobalConfig.get();
+  const globbedFiles = await globAsync(upath.join(localDir, globPattern));
```

Comment:
I would prefer to use `git.getFiles` and minimatch instead.

@rarkins WDYT?

Reply:
I can't find the `git.getFiles` function. Would you mind sending me a link?

Reply:
https://github.com/renovatebot/renovate/blob/f5cc1de36f7d6f3ce7d689a9b6675a930d81337c/lib/util/git/index.ts#L476